master/buildbot/util/git.py

```diff
@@ -75,14 +75,11 @@ def setupGit(self, logname=None):
         self.supportsSshPrivateKeyAsConfigOption = False
 
     def parseGitFeatures(self, version_stdout):
-
-        if 'git' not in version_stdout:
+        match = re.match("^git version (\\d+(\\.\\d+)*)", version_stdout)
```

Comment:
raw string would be better: `r"^git version (\d+(\.\d+)*)"`

Reply:
thanks, I've updated it accordingly